handycon.py

```diff
@@ -746,15 +747,16 @@ async def capture_power_events():
                     active_keys = keyboard_device.active_keys()
                     if event.type == e.EV_KEY and event.code == 116: # KEY_POWER
                         if event.value == 0:
+                            steam_path = HOME_PATH / '.steam/root/ubuntu12_32/steam'
                             if active_keys == [125]:
                                 # For DeckUI Sessions
                                 shutdown = True
-                                cmd = 'su {} -c "{}/.steam/root/ubuntu12_32/steam -ifrunning steam://longpowerpress"'.format(USER, HOME_PATH)
+                                cmd = f'su {USER} -c "{steam_path} -ifrunning steam://longpowerpress"'
```

Comment:
Did you test this? short press should show an animation and long press should pop up the power menu.

Reply:
Short press put my machine to sleep (expected desktop environment behavior).  Long press forced shutdown (expected BIOS behavior).

Reply:
Was there an animation for short press? I have a catch for regular suspend if we're in desktop mode, but steam does some magic that makes resuming a game in progress more stable.


"Long press" is about 2 seconds